hw4/Task_9.ts

```diff
@@ -0,0 +1,15 @@
+//Task 9
+// Дано произвольное число.
+// Реализовать метод проверяющий является ли это число целочисленным или нет.
+// Если является - вывести его, если нет - округлить до целой части.
+
+function myFunction9(myNumber1: number): number {
+    if (Number.isInteger(myNumber1)) {
+        console.log(myNumber1)
+    } else {
+        console.log(Math.ceil(myNumber1))
+    }
+    return (myNumber1)
```

Comment:
no need to return here

Reply:
If I remove it, it throws an error:  'A function whose declared type is neither 'void' nor 'any' must return a value'.